src/ListView.vala

```diff
@@ -30,20 +30,30 @@ public class Tasks.ListView : Gtk.Grid {
         label_style_context.add_class (Granite.STYLE_CLASS_H1_LABEL);
         label_style_context.add_class (Granite.STYLE_CLASS_ACCENT);
 
-        var settings_button = new Gtk.Button.from_icon_name ("view-more-horizontal-symbolic", Gtk.IconSize.MENU);
+        var settings_button = new Gtk.ToggleButton ();
```

Comment:
Instead of a ToggleButton, can we use a MenuButton here? Then we won't have to manually activate and deactivate the button and manually hide/show the popover

Reply:
You are right.